rewording (#402)
ietf-wg-gnap/gnap-core-protocol: draft-ietf-gnap-core-protocol.md

Dynamically generated client instance identifiers are string values that MUST be
protected by the client instance as secrets. Instance identifier values MUST be unguessable

> and MUST NOT contain any information that would compromise the AS if revealed. Instance identifier values are

Reviewer: ```suggestion
and MUST NOT contain any information that would compromise the system if revealed. Instance identifier values are
```

> and MUST NOT contain any information that would compromise the AS if revealed. Instance identifier values are

Reviewer: ```suggestion
and MUST NOT contain any information that would compromise any party if revealed. Instance identifier values are
```
